test/yaml_elixir_test.exs

```diff
@@ -205,10 +205,21 @@ defmodule YamlElixirTest do
     assert {:error, %YamlElixir.ParsingError{}} = YamlElixir.read_from_string(yaml)
   end
 
+  test "exception structs should have line, column, and type values" do
+    yaml = "*invalid"
```

Comment:
There's no need to introduce this variable here. You can pass it as a string literal directly into the function below.